apps/src/applab/sanitizeHtml.js

```diff
@@ -74,6 +80,9 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
           attribs: attribs
         };
       }
+    },
+    exclusiveFilter: function (element) {
+      return (ignoredTags.indexOf(element.tag) !== -1);
```

Comment:
Does this filter need to be case-insensitive, or is element.tag guaranteed to be lowercase?  Might be worth covering with a test.